h3/h3.py

```diff
@@ -759,9 +762,19 @@ def h3_distance(h3_address_origin, h3_address_h3):
         string_to_h3(h3_address_origin), string_to_h3(h3_address_h3))
 
 
+def h3_line_size(h3_address_origin, h3_address_h3):
+    return libh3.h3LineSize(string_to_h3(h3_address_origin), string_to_h3(h3_address_h3))
+
+
 def h3_line(h3_address_origin, h3_address_h3):
-    array_len = h3_distance(h3_address_origin, h3_address_h3) + 1
+    array_len = h3_distance(h3_address_origin, h3_address_h3)
```

Comment:
This line should be switched to h3LineSize

Reply:
good catch